cmake/packages_check.cmake

```diff
@@ -18,6 +20,14 @@ elseif(COMPILER_TOOLCHAIN STREQUAL "clang")
     set(ffmpeg_hardcoded_tables "--enable-hardcoded-tables")
     set(mpv_lto_mode "-Db_lto_mode=thin")
     set(mpv_copy_debug COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.pdb ${CMAKE_CURRENT_BINARY_DIR}/mpv-debug/mpv.pdb)
+    if(CLANG_PACKAGES_LTO)
+        set(lto_rustflags "-C linker-plugin-lto=yes -C embed-bitcode=yes -C lto=thin")
+        set(ffmpeg_mlp "--disable-decoder=mlp --disable-encoder=mlp --disable-demuxer=mlp --disable-muxer=mlp --disable-parser=mlp")
+        set(x264_lto "--enable-lto")
+        if(GCC_ARCH_HAS_AVX)
```

Comment:
Why the long check for GCC_ARCH_HAS_AVX though?

Reply:
So, this disable lto for compiler which has AVX define? x86-64 target doesnt has avx, only march=native. There should be a note here to explain